PxCs/Data/Sound/PxSoundData.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+
+namespace PxCs.Data.Sound
+{
+    public class PxSoundData<T>
+    {
+        public Type arrayType;
+        public List<T> sound;
```

Comment:
Please use an array for performance reasons as the data size is fixed. All other PxCs data uses arrays too. On unity side we leverage dynamic lists.